### ISSUE TITLE
ENAMETOOLONG: yarle should truncate note titles that exceed maximum filename length

### ISSUE THREAD

**Issue by ecerulm** [akosbalasko/yarle#462]
`yarle` failed iwht `ENAMETOOLONG` on macOS, one of the notes has a really long title /name and `yarle` tries to create a filename that exceeds 255 character (which I believe is the maximum) . 

I have a few of those notes, so for me it would be great if `yarle` could automatically truncate the filename (while keeping the links to it working). 

```
node:internal/fs/utils:346
    throw err;
    ^

Error: ENAMETOOLONG: name too long, open '/Users/ecerulm/Downloads/ObsidianVault/software/notes/dataengineering/ClientError_ An error occurred (AccessDenied) when calling the DescribeClusters operation_ User_ arn_aws_sts__xxxxxxxxxxx_assumed-role_aihub-service-account-role_xxxx is not authorized to perform_ redshift_DescribeClusters on resource_ arn_aws_redshift_e.md'
    at Object.openSync (node:fs:585:3)
    at Object.readFileSync (node:fs:453:35)
    at EventEmitter.<anonymous> (/Users/ecerulm/.npm/_npx/106a035a6ccf201d/node_modules/yarle-evernote-to-md/dist/yarle.js:137:54)
    at EventEmitter.emit (node:events:512:28)
    at SAXStream.<anonymous> (/Users/ecerulm/.npm/_npx/106a035a6ccf201d/node_modules/xml-flow/lib/xml-flow.js:142:15)
    at SAXStream.emit (node:events:512:28)
    at me._parser.<computed> [as onclosetag] (/Users/ecerulm/.npm/_npx/106a035a6ccf201d/node_modules/sax/lib/sax.js:258:17)
    at emit (/Users/ecerulm/.npm/_npx/106a035a6ccf201d/node_modules/sax/lib/sax.js:624:35)
    at emitNode (/Users/ecerulm/.npm/_npx/106a035a6ccf201d/node_modules/sax/lib/sax.js:629:5)
    at closeTag (/Users/ecerulm/.npm/_npx/106a035a6ccf201d/node_modules/sax/lib/sax.js:889:7) {
  errno: -63,
  syscall: 'open',
  code: 'ENAMETOOLONG',
  path: '/Users/ecerulm/Downloads/ObsidianVault/software/notes/dataengineering/ClientError_ An error occurred (AccessDenied) when calling the DescribeClusters operation_ User_ arn_aws_sts__xxxxx_assumed-role_aihub-service-account-role_xxxx is not authorized to perform_ redshift_DescribeClusters on resource_ arn_aws_redshift_e.md'
}


```

**Comment by akosbalasko** [comment]
Hi @ecerulm ,
thanks for reporting this issue, I have a feeling that I've already implemented it in a PR, but forgot to merge it to the master. Let me double check it, I'll keep you informed.

**Comment by akosbalasko** [comment]
Hi @ecerulm ,
I merged the PR and released the fix as 5.0.11, so the latest version should solve this problem. Could you please confirm that it works? 
Thank you very much!

**Comment by ecerulm** [comment]
yeah, that worked. I'm no longer getting the `ENAMETOOLONG`
